apps/hash-ai-worker-ts/src/activities/flow-activities/research-entities-action/deduplicate-entities.ts

```diff
@@ -35,7 +35,9 @@ export const deduplicationAgentSystemPrompt = `
   3. the same or very similar name may refer to different entities
   4. the same or very similar summary may refer to different entities
 
-  If in doubt, do not merge entities. If you are confident that two entities are duplicates, merge them.
+  If an entity is a version of another entity, do not report it as a duplicate as they are not referring to the same thing.
```

Comment:
I don't think it's clear here what 'a version of another entity' means (I only know because of the context of the LLM model example). Whether a newer version of something is a duplicate of something else is probably also use case dependent. I've tried to clarify a bit.

```suggestion
  If an entity is a different version of another entity (e.g. a different version of a piece of software, a newer model of a car, or the Toastmaker 3000 versus the Toastmaker 2000), do not report it as a duplicate as they are not referring to the same thing.
```